index.html

```diff
@@ -61,6 +61,205 @@ <h3>LET'S CONNECT</h3>
+            <li>
+              <img src="images/works_images/Counter.png" alt="dot icon" />
+            </li>
+            <li>Back End Dev</li>
+            <li>
+              <img src="images/works_images/Counter.png" alt="dot icon" />
+            </li>
+            <li>2015</li>
+          </ul>
+          <p class="works-card--detail">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="works-card--languages">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button class="portfolio-button">See Project</button>
+        </div>
+
+        <div class="works-card">
+          <img
+            class="works-card--img"
+            src="images/works_images/Snapshoot_Portfolio3.png"
+            alt="project pic"
+          />
+          <h1 class="works-card--title">Tonic</h1>
+          <ul class="works-card--info">
+            <li class="canopy">CANOPY</li>
+            <li>
+              <img src="images/works_images/Counter.png" alt="dot icon" />
+            </li>
+            <li>Back End Dev</li>
+            <li>
+              <img src="images/works_images/Counter.png" alt="dot icon" />
+            </li>
+            <li>2015</li>
+          </ul>
+          <p class="works-card--detail">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="works-card--languages">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button class="portfolio-button">See Project</button>
+        </div>
+
+        <div class="works-card">
+          <img
+            class="works-card--img"
+            src="images/works_images/Snapshoot_Portfolio4.png"
+            alt="project pic"
+          />
+          <h1 class="works-card--title">Multi-Post Stories</h1>
+          <ul class="works-card--info">
+            <li class="canopy">CANOPY</li>
+            <li>
+              <img src="images/works_images/Counter.png" alt="dot icon" />
+            </li>
+            <li>Back End Dev</li>
+            <li>
+              <img src="images/works_images/Counter.png" alt="dot icon" />
+            </li>
+            <li>2015</li>
+          </ul>
+          <p class="works-card--detail">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="works-card--languages">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button class="portfolio-button">See Project</button>
+        </div>
+      </section>
+
+      <section class="about-myself-section">
+        <div class="about-myself--info">
+          <h1 class="about-myself--info--title">About Myself</h1>
```

Comment:
- [ ] Kindly consider use of `<h1>` tag once in a a page. This is because for [Semantic HTML](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwisnebC8Mb8AhX-8bsIHZBQArsQFnoECAsQAw&url=https%3A%2F%2Fdeveloper.mozilla.org%2Fen-US%2Fdocs%2FGlossary%2FSemantics&usg=AOvVaw1B6qsXIooodDa3t1Uxayp1) purposes the ` <h1>` element is a semantic element, which gives the text it wraps around the role( or meaning) of a top level heading on your page. For this case you may consider using `<h2> - <h6>` elements.